tests/test_locking.py

```diff
@@ -28,6 +28,7 @@ def test_create_and_update_lock_files(
             platforms:
             - win-64
             - linux-64
+            - osx-arm64
             dependencies:
             - pytest
             - wincom  # [win]
```

Comment:
Will be nice to have a line selectors (like `- wincom # [win]` and `- shmem # [unix]`) to check if `osx` and/or `osx-arm64` is being properly applied to those.